File: Beginner/Chef and SnackDown (SNCKYEAR)/snackdown.cpp
#include <iostream>
using namespace std;

int main() {
	int a ;
	int b ; 
	cin >> a ; 
	while(a--){
	   cin >> b ;
	   if(b == 2010 || b == 2015 || b == 2016 || b == 2017  || b == 2019){
	       cout << "HOSTED"<< endl ;
	   }
	   else {
	       cout << "NOT HOSTED"<< endl ;
	   }
	}
	return 0;
}

File: Beginner/Chef and SnackDown (SNCKYEAR)/snackdown.java
import java.util.*;
import java.lang.*;
import java.io.*;


class Codechef {
	public static void main (String[] args) throws java.lang.Exception {
		Scanner scan=new Scanner(System.in);
		if(scan.hasNext()){
		    int t = scan.nextInt();
		    for(int i=0; i<t; i++){
		        int num = scan.nextInt();
		        if(num==2010||num==2015||num==2016||num==2017||num==2019)
		            System.out.println("HOSTED");
		        else
		            System.out.println("NOT HOSTED");
		    }
        }
        scan.close();
	}
}